SINGE_Example.m

```diff
@@ -0,0 +1,47 @@
+%% Simple example that runs SCINGE for two replicates of two hyperparameter settings
+clear all;
+close all;
+clc;
+if ~isdeployed
+    addpath(genpath('.'));
+end
+
+%% Import list of parameter combinations
+fid = fopen('SINGE_params.cfg');
```

Comment:
We should support specifying the config file names as a command line argument.  This could be the default name or we could require both types of config files to be provided before running.  That way we can compile this file one and let users run on different datasets without needing to recompile.